salt/modules/opkg.py

```diff
@@ -435,6 +435,15 @@ def remove(name=None, pkgs=None, **kwargs):  # pylint: disable=unused-argument
         A list of packages to delete. Must be passed as a python list. The
         ``name`` parameter will be ignored if this option is passed.
 
+    remove_dependent_packages
```

Comment:
Maybe "remove_dependencies" would be better. What else than "packages" can be possibly removed in the context of "dependency"?